src/test/java/seedu/sellsavvy/logic/parser/AddressBookParserTest.java

```diff
@@ -51,8 +51,14 @@ public class AddressBookParserTest {
     @Test
     public void parseCommand_addPerson() throws Exception {
         Person person = new PersonBuilder().build();
+
+        // using command word
         AddPersonCommand command = (AddPersonCommand) parser.parseCommand(PersonUtil.getAddPersonCommand(person));
         assertEquals(new AddPersonCommand(person), command);
+
+        // using command alias
+        command = (AddPersonCommand) parser.parseCommand(PersonUtil.getAddPersonCommandAlias(person));
```

Comment:
Just a small qn why this one use PersonUtil but there rest get directly from command

Reply:
in the default implementation, PersonUtil only includes `getAddPersonCommand` and `getEditPersonDescriptorDetails`, i'm guessing because addPerson command and editPerson command has more parameters.
Currently am just following what is being done in the initial implementation.